wp-content/themes/core/assets/pcss/layout/_variables.pcss

```diff
@@ -11,5 +11,8 @@
 body {
 	--grid-margin: var(--wp--custom--spacing--grid-margin);
 	--grid-gutter: var(--wp--custom--spacing--grid-gutter);
-	--grid-max-width: var(--wp--style--global--wide-size);
+	--grid-wide-width: var(--wp--style--global--wide-size);
+
+	/* this should match the "Grid" size defined in theme.json */
+	--grid-max-width: 1680px;
```

Comment:
I wonder if we should programmatically create some css custom props based on the sizes we define in the tribe plugin?

At least for now, maybe name this something like `--tribe--style--global--grid-size` so it matches Core's global style names, but uses a custom namespace? I suppose to match the plugin name something like `--beca--style...` would make sense.

I'd also make this comment a little clearer that these are the sizes defined in the `_experimentalLayout` section as used by our plugin.